Makefile

```diff
@@ -19,34 +20,26 @@ docs-down:
 	(cd docs && docker compose down)
 
 .PHONY:dev
-dev:beaker_kernel/server/dev_ui/build/index.js
-	if [[ "$$(docker compose ps | grep 'jupyter')" == "" ]]; then \
-		docker compose pull; \
-		docker compose up -d --build && \
-		(sleep 1; python -m webbrowser "http://localhost:8888/dev_ui"); \
-		docker compose logs -f jupyter || true; \
-	else \
-		docker compose down jupyter && \
-		docker compose up -d jupyter && \
-		(sleep 1; python -m webbrowser "http://localhost:8888/dev_ui"); \
-		docker compose logs -f jupyter || true; \
-	fi
+dev:
+	docker compose up -d --build && \
+	(sleep 1; python -m webbrowser "http://localhost:8888/"); \
+	docker compose logs -f jupyter || true; \
 
 .env:
 	@if [[ ! -e ./.env ]]; then \
 		cp env.example .env; \
 		echo "Don't forget to set your OPENAI key in the .env file!"; \
 	fi
 
-beaker_kernel/server/dev_ui/node_modules:beaker_kernel/server/dev_ui/package*.json
+beaker-vue/node_modules:beaker-vue/package*.json
 	export `cat .env` && \
-	(cd beaker_kernel/server/dev_ui && npm install) && \
-	touch beaker_kernel/server/dev_ui/node_modules
+	(cd beaker-vue && npm install) && \
+	touch beaker-vue/node_modules
 
-beaker_kernel/server/dev_ui/build/index.js:beaker_kernel/server/dev_ui/node_modules beaker_kernel/server/dev_ui/src/** beaker_kernel/server/dev_ui/index.css beaker_kernel/server/dev_ui/*.js beaker_kernel/server/dev_ui/*.json beaker_kernel/server/dev_ui/templates/**
-	export `cat .env` && \
-	(cd beaker_kernel/server/dev_ui && npm run build) && \
-	touch beaker_kernel/server/dev_ui/build/*
+beaker_kernel/server/ui/index.html:beaker-vue/node_modules beaker-vue/**
```

Comment:
Quick question on the dependencies for this task (I don't use make that often):
Does it need to have "dependencies" on bth node_modules and beaker-vue/** ? Would the latter rule cover the first one? Or, would package-lock.json changing suffice?

I suppose I'll search for answers, but something very small that stood out to me (and I don't how it works)